obs/window-basic-main.cpp

```diff
@@ -80,6 +80,11 @@ OBSBasic::OBSBasic(QWidget *parent)
 {
 	ui->setupUi(this);
 
+	QFile File("stylesheet.qss");
+	File.open(QFile::ReadOnly);
+	QString StyleSheet = QLatin1String(File.readAll());
```

Comment:
Why use ```QLatin1String```? ```QString str = file.readAll();``` should automatically produce UTF8, which is probably more desirable than other encodings?